### PR TITLE
Fix feature selection + filter transforms

### PR DESCRIPTION
**Pull request by alex-hse-repository**
## Before submitting (must do checklist)

- [ ] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [ ] Did you update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [ ] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

### CODE REVIEW

File: tests/test_transforms/test_feature_selection/test_feature_importance_transform.py [hunk 1]
```diff
@@ -239,17 +241,18 @@ def test_sanity_model(model, ts_with_regressors):
...
 @pytest.mark.parametrize("top_k", [0, 1, 5, 15, 50])
 def test_mrmr_right_len(relevance_table, top_k, ts_with_regressors):
     """Check that transform selects exactly top_k regressors."""
-    df = ts_with_regressors.to_pandas()
-    mrmr = MRMRFeatureSelectionTransform(relevance_table=relevance_table, top_k=top_k)
-    df_selected = mrmr.fit_transform(df)
     all_regressors = ts_with_regressors.regressors
+    ts = ts_with_regressors
+    mrmr = MRMRFeatureSelectionTransform(relevance_table=relevance_table, top_k=top_k)
+    df_selected = mrmr.fit_transform(ts)
```
Comment (Mr-Geekman) [inline tests/test_transforms/test_feature_selection/test_feature_importance_transform.py:254]: `df_selected`? Not `ts_selected`?

Comment (Mr-Geekman) [reply tests/test_transforms/test_feature_selection/test_feature_importance_transform.py:254]: May be `to_pandas()` should be added.

Comment (alex-hse-repository) [reply tests/test_transforms/test_feature_selection/test_feature_importance_transform.py]: Fixed

Comment (alex-hse-repository) [reply tests/test_transforms/test_feature_selection/test_feature_importance_transform.py]: Fixed

**Review by Mr-Geekman — APPROVED** [review]

**Pull request merged**